pep-0570.rst

```diff
@@ -149,13 +203,12 @@ Providing positional-only arguments will allow for maintaining the
 interface when creating pure Python implementation of C modules, which
 provides not only the API benefits outlined in this document but it is
 also faster. See this thread about converting keyword arguments to positional:
-[#thread-keyword-to-positional]_ and PEP-399 [#PEP399]_, which requires the
-same API for C accelerators as the Python implementation.
+[#thread-keyword-to-positional]_.
 
 There have been multiple changes in builtin functions that moved away
 from keyword arguments, like ``bool``, ``float``, ``list``, ``int``, ``tuple``
-which is a non-backward compatible change. By having proper support for
-positional-only arguments, these kind of APIs, where it is clear that
+which is a non-backwards compatible change. By having proper support for
+positional-only arguments, these kinds of APIs, where it is clear that
 passing a keyword argument provides no clarity, it would be possible to
 follow a similar approach as these builtins, without breaking users.
```

Comment:
This part is a bit unclear to me. The earlier part of the paragraph mentions making backwards-incompatible changes, but this is talking about non-breaking changes.

I also think this is another example where you are mixing arguments in favor of using positional-only arguments *at all*, and arguments in favor of adding Python syntax for them. Perhaps this paragraph and the previous one can be reworked along those lines, e.g.:

>In addition to the API benefits outlined earlier in this document, positional-only arguments are also faster, as demonstrated in this thread about converting keyword arguments to positional: [#thread-keyword-to-positional]_. In fact, because of these benefits there has even been a recent trend towards moving builtins *away* from keyword arguments: recently, backwards-incompatible changes were made to disallow keyword arguments to ``bool``, ``float``, ``list``, ``int``, ``tuple``.

> Providing a way to specify positional-only arguments in Python will make it easier to maintain pure Python implementations of C modules and will allow users to take advantage of these benefits even in code written only in Python. It will also encourage users to start with positional-only arguments when they believe that passing a keyword argument provides no clarity; unlike making a keyword argument positional-only, allowing a positional argument to be passed positionally is not a breaking change.

Not sure how strong that second paragraph is, but I think you can see the way I'm trying to arrange the argument.